packages/common-all/src/utils.ts

```diff
@@ -1,10 +1,11 @@
-import minimatch from "minimatch";
-import semver from "semver";
 import GithubSlugger from "github-slugger";
-import querystring from "querystring";
 import _ from "lodash";
+import minimatch from "minimatch";
+import querystring from "querystring";
```

Comment:
I think I got warned installing that `querystring` is [deprecated](https://www.npmjs.com/package/querystring)